### PR TITLE
V1 docs audit

### PR DESCRIPTION
**Pull request by bryanhuhta**
I ran through the v1 docs and came up with a number of minor corrections.

### CODE REVIEW

File: docs/sources/configure-server/about-tenant-ids.md [hunk 1]
```diff
@@ -14,11 +14,11 @@ Grafana Pyroscope is a multi-tenant system where tenants can query profiles that
 Within a Grafana Pyroscope cluster, the tenant ID is the unique identifier of a tenant.
 The query takes the tenant ID from the `X-Scope-OrgID` parameter that exists in the HTTP header of each request, for example `X-Scope-OrgID: <TENANT-ID>`.
 
-To push profiles to Grafana Pyroscope for a specific tenant refer to [Configure the Agent]({{<relref "../configure-client/_index.md">}}).
+To push profiles to Grafana Pyroscope for a specific tenant, refer to [Configure the Agent]({{<relref "../configure-client/_index.md">}}).
 
 ## Restrictions
 
-Tenant IDs must be less-than or equal-to 150 bytes or characters in length and can only include the following supported characters:
+Tenant IDs cannot be longer than 150 bytes or characters in length and can only include the following supported characters:
```
Comment (petethepig) [inline docs/sources/configure-server/about-tenant-ids.md:21]:
```suggestion
Tenant IDs cannot be longer than 150 characters in length and can only include the following supported characters:
```

**Review by petethepig — APPROVED** [review]
Looks great, thanks for doing all of this!

File: docs/sources/reference-pyroscope-architecture/block-format/_index.md [hunk 2]
```diff
@@ -13,6 +13,8 @@ This document describes how Pyroscope stores the data in its blocks. Each
 block belongs to a single tenant and is identfied by a unique [ULID]. Within
```
Comment (kolesnikovae) [inline docs/sources/reference-pyroscope-architecture/block-format/_index.md:13]:
```suggestion
block belongs to a single tenant and is identified by a unique [ULID]. Within
```

**Comment by kolesnikovae** [comment]
Looks great! I'm currently working on the updated documentation of the block format (nothing special, just want to reflect the changes we made). It would be great if you could review it as well :)

**Pull request merged**